PubHub/PubHub.AdminPortal/Components/Constants/ValidationAnnotationConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace PubHub.AdminPortal.Components.Constants
+{
+    public class ValidationAnnotationConstants
+    {
+        public const int MAX_NAME_LENGTH = 128;
+        public const int MAX_EMAIL_LENGTH = 256;
+        public const int MAX_PASSWORD_LENGTH = 64;
+        public const int MAX_TITLE_LENGTH = 150;
+        public const int MAX_SUMMARY_LENGTH = 2000;
+    }
```

Comment:
These could be put into `Common` and be shared across API, Mobile and Portal 🤔

Reply:
It was just because you had yours in the mobile but, I can move it - then you can modify to use these if you'd like in the mobile.

Reply:
I have moved it to the common library 😊